src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entries.html.twig

```diff
@@ -183,4 +183,39 @@
     </div>
     {% endif %}
 
+    <!-- Sort -->
+    <div id="sort" class="side-nav fixed right-aligned">
+
+        <h4 class="center">{{ 'entry.sort.title'|trans }}</h4>
+
+        <div class="row">
+            <div class="col s12">
+                <label>{{ 'entry.sort.status_label'|trans }}</label>
+            </div>
+
+            <div class="input-field col s12 with-checkbox">
+                <select>
+                    <option value="" disabled selected>Creation date</option>
+                    <option value="createdAt">{{ 'entry.sort.by.creation_date' | trans }}</option>
+                    <option value="title">{{ 'entry.sort.by.title' | trans }}</option>
+                    <option value="updatedAt">{{ 'entry.sort.by.last_updated' | trans }}</option>
+                </select>
+            </div>
+
+            <div class="col s12">
+                <div class="switch">
+                    <label>
+                        Ascendant
+                        <input type="checkbox">
+                        <span class="lever"></span>
+                        Descendant
+                    </label>
+                </div>
+            </div>
+        </div>
+
+    </div>
+
+    {% include "WallabagCoreBundle:Entry:pager.html.twig" with {'entries': entries} %}
```

Comment:
@tcitworld that file is missing from your PR. It block the PR as I guess only you has it.

Reply:
I very probably lost it.